src/D2L.Security.OAuth2/Keys/Default/EcDsaJsonWebKey.ECCPublicKeyBlobFormatter.cs

```diff
@@ -1,5 +1,6 @@
 ﻿using System;

```

Comment:
I _believe_ they've added stuff that we could migrate to and make this unnecessary, will assess after.